### PR TITLE
Update loftwah.json

### PR DESCRIPTION
**Pull request by loftwah**
## Changes proposed

Add a new link and also a new milestone

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Update my profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4764"></a>

### CODE REVIEW

File: data/loftwah.json [hunk 1]
```diff
@@ -202,6 +207,14 @@
       "color": "#EF5904",
       "description": "I accepted the role as EddieHub Ambassador 0.2",
       "url": "https://www.eddiehub.org/ambassador-program"
+    },
+    {
+      "title": "Received my first GitHub Sponer",
```
Comment (SaraJaoude) [inline data/loftwah.json:212]: Just wondered if you meant `Sponsor` rather than Sponer

Comment (loftwah) [reply data/loftwah.json:212]: Hahaha great pickup

Comment (SaraJaoude) [reply data/loftwah.json:212]: I didn't want to assume it wasn't an Australian term... 😜

Comment (loftwah) [reply data/loftwah.json]: No, I'm just a little bit special 😅 I have fixed it. No more GitHub Sponers for me.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**